04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,83 @@
+const Product = require("../models/product");
+//////////////////////////////////////////////////////////////////STATIC/////////////////////////////////////////////////////////////////////////////
+
+const getAllProductsStatic = async (req, res) => {
+  const search = "a";
+  const products = await Product.find({ price: { $gt: 100 } })
+    .sort("name")
+    .select("name price")
+    .limit(10)
+    .skip(1).numericFilters;
```

Comment:
Requires change
The `search` variable is not used and not needed, therefore it can be removed. Also, I believe `.numericFilters` after `.skip(1)` is not valid and should be removed as well.